src/app/lib/utilities/preprocessor/rules/filterContentType.js

```diff
@@ -17,12 +17,12 @@ const filterUnknownContentTypes = data => {
     if (Array.isArray(group.items)) {
       newGroup.items = group.items.filter(item => {
         const itemType = item.assetTypeCode || item.cpsType;
-        const validItemType = whitelist.includes(itemType);
-        const validContentType =
-          item.assetTypeCode !== 'PRO' ||
-          contentTypes.includes(item.contentType);
-
-        return itemType && validItemType && validContentType;
+        const isValidItemType = whitelist.includes(itemType);
```

Comment:
I agree that this labelling with constants is much clearer way of writing the logic for whether to render useful links.  
I believe it'd be better to think about how we can unit test these rules more thoroughly, so that we don't need to have the complex rules replicated in the Cypress tests above. 
Since this work is preventing Cypress tests on CD from passing, I'll be happy to let it go through. 

This can be an example case where we could replace cypress end-to-end tests that run against all services with unit tests and other levels of business-led integration tests that do not need to be run against every single service. https://github.com/bbc/simorgh/issues/4486.